fhir-server/liberty-config/server.xml

```diff
@@ -50,7 +50,7 @@
         
         To send the trace messages to standard out, set TRACE_FILE to "stdout".
     -->
-    <variable name="TRACE_SPEC" defaultValue="*=info"/>
```

Comment:
we probably don't want finer by default, right?

Reply:
Correct, I'm still working through the failure with reindex, and needed more detailed logs as I setup db2 locally.